src/services/formatting/formatting.ts

```diff
@@ -445,7 +445,7 @@ namespace ts.formatting {
                     if ((<MethodDeclaration>node).asteriskToken) {
                         return SyntaxKind.AsteriskToken;
                     }
-                    // fall-through
+                // fall-through

```

Comment:
Also a result of auto-formatting. Should we make an exception for comments within `switch`/`case`?